sdk/storage/storage-file-share/src/ShareFileClient.ts

```diff
@@ -1395,31 +1396,36 @@ export class ShareFileClient extends StorageClient {
   // High Level functions
 
   /**
-   * ONLY AVAILABLE IN BROWSERS.
```

Comment:
A similar method with the same name `uploadBrowserData` is present in storage-blob.
What to do for blob?

Reply:
We can add `uploadDate`, and mark `uploadBrowserData` as @deprecate

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/6120
Logged an issue.